deploy/cr.yaml

```diff
@@ -18,17 +18,20 @@ spec:
   nodeSelector: {}
   # https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ (optional)
   tolerations:
-  - effect: NoSchedule
-    key: node-role.kubernetes.io/master
-    operator: Exists
+    - effect: NoSchedule
+      key: node-role.kubernetes.io/master
+      operator: Exists
   # oneagent installer image (optional)
   # certified image from Red Hat Container Catalog for use on OpenShift: registry.connect.redhat.com/dynatrace/oneagent
   # for kubernetes it defaults to docker.io/dynatrace/oneagent
-  #image: ""
+  image: ""
```

Comment:
Comment it again. I believe I commented it because it was affecting the CSV file generation.

Reply:
Done